docs/question/test-cycle.d2

```diff
@@ -0,0 +1,39 @@
+direction: down
+# Shapes
+generate: "generate()" {
+  tooltip: "Generate random parameters and correct answers"
```

Comment:
Aside: the UX for these tooltips seems a little funky. My first instinct was to click the `(i)` icon, but that just opened the diagram in a new tab.

<img width="353" alt="Screenshot 2025-05-05 at 09 03 35" src="https://github.com/user-attachments/assets/1f95121d-9674-4056-b5ab-167d683a55eb" />

Reply:
I think https://github.com/terrastruct/d2/issues/897 is exactly the feature request that we want. Let's keep an eye on that.

In the meantime, maybe we can avoid using tooltips entirely? This PR isn't introducing them for the first time, so I'm happy to land this as-is and then worry about replacing all usages later if that's what we decide to do. From what I can see, `docs/question/lifecycle.d2` is the only other place they're used.

Reply:
Sounds good.